deeppavlov/core/agent/rich_content.py

```diff
@@ -97,6 +100,10 @@ class RichMessage(RichItem):
     def __init__(self) -> None:
         self.controls: list = []
 
+    def __str__(self, *args, **kwargs):
+        result = '\n'.join([str(control) for control in self.controls if str(control)])
+        return result or ''
```

Comment:
```suggestion
        return result
```
`'\n'.join([])` will already return `''`